packages/pageflow/src/frontend/audio/audio.js

```diff
@@ -1,11 +1,15 @@
 import _ from 'underscore';
```

Comment:
Having both `audio/index.js` and `audio/audio.js` was a workaround since there was no way to assign `pageflow.Audio`  in `audio/index.js` and _then_ import `audio/multi_player.js` which assigns `pageflow.Audio.MultiPlayer`. With real import/export that is no longer needed. There only needs to be `Audio/index.js` now, which can export `Audio` and import `./MultiPlayer`.

Reply:
will it not cause circular dependency ? 
e.g. when Lazy.js imports VideoPlayer from same index?

Reply:
Yeah, but I just realized that putting `export * from './lazy'` in `videoPlayer/index.js` is an error anyway. Doesn't this cause `pageflow.Lazy` to be defined instead of `pageflow.VideoPlayer.Lazy`?

Reply:
As discussed in the call. Exports are handled in such a way that Internal files are assigned to
external components which gets exported.
Forexample
pageflow.VideoPlayer will have Lazy assigned to it.